CMakeLists.txt

```diff
@@ -101,7 +101,7 @@ ExternalProject_Add(
     ext_ngraph
     GIT_REPOSITORY https://github.com/NervanaSystems/ngraph
     GIT_TAG v0.5.0
-    CMAKE_ARGS -DNGRAPH_INSTALL_PREFIX=${CMAKE_CURRENT_BINARY_DIR}/ngraph/ngraph_dist -DNGRAPH_USE_PREBUILT_LLVM=${NGRAPH_USE_PREBUILT_LLVM}
+    CMAKE_ARGS -DNGRAPH_INSTALL_PREFIX=${CMAKE_CURRENT_BINARY_DIR}/ngraph/ngraph_dist -DNGRAPH_USE_PREBUILT_LLVM=${NGRAPH_USE_PREBUILT_LLVM} -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
```

Comment:
As a Fan of `gdb` I need to have this option available 😉

Reply:
Me too 👍 Can we split this long line to multiple lines?